ts/modules/boost/wgl-renderer.ts

```diff
@@ -1645,7 +1645,7 @@ function GLRenderer(
         shader.destroy();
         if (gl) {
 
-            objectEach(textureHandles, function (key: string): void {
+            objectEach(textureHandles, function (_val, key): void {
                 if (textureHandles[key].handle) {
```

Comment:
Now that we have `val` passed in, we can probably use that directly. `if (val.handle) {` etc.